apps/company/views.py

```
@@ -122,6 +171,660 @@ class CompanyStocksView(APIView):
+            supplier = RecomamandationSupplier.objects.filter(company=company, product__vendor_code__contains=article,marketplace_type__icontains=service).distinct("product")
+        supplier = supplier.filter(warehouse__region_name__contains=region_name)
+        if not supplier.exists():
+            supplier = supplier.filter(warehouse__oblast_okrug_name__contains=region_name)
+        context = {"market": service}
+        
+        paginator = Paginator(supplier, per_page=page_size)
+        page = paginator.get_page(page)
+        count = paginator.count
+        serializer = RecomamandationSupplierSerializer(page, context=context, many=True)
+        if sort and sort in ["-1", "1"]:
+            ordering_by_quantity = False if sort =="-1" else True
```

Ruff

Use `not ...` instead of `False if ... else True`

Replace with `not ...`

(SIM211)